src/System.Net.Security/src/System/Net/SecureProtocols/NegoState.cs

```diff
@@ -838,20 +776,26 @@ private static void ReadCallback(IAsyncResult transportResult)
             }
         }
 
-        private unsafe byte[] GetOutgoingBlob(byte[] incomingBlob, ref Win32Exception e)
+        internal static bool IsError(SecurityStatusPal status)
         {
-            Interop.SecurityStatus statusCode;
+            return ((int)status.ErrorCode >= (int)SecurityStatusPalErrorCode.OutOfMemory);
```

Comment:
Defined here, this has the risk that if the `SecurityStatusPalErrorCode` enum changes and this doesn't, the code will break.
My recommendation is to keep the `IsError` part of the `SecurityStatusPal` class, near the enum.